lib/vpi/VpiImpl.cpp

```diff
@@ -215,6 +212,8 @@ GpiObjHdl* VpiImpl::create_gpi_obj_from_handle(vpiHandle new_hdl,
 
 GpiObjHdl* VpiImpl::native_check_create(void *raw_hdl, GpiObjHdl *parent)
 {
+    GpiObjHdl *parent_hdl = sim_to_hdl<GpiObjHdl*>(parent);
```

Comment:
So the reason I did this was because in VPI and VHPI the function `native_check_create(int32_t index, GpiObjHdl *parent)` was doing that `sim_to_hdl<>()` call.  Frankly I didn't understand why it was there, but I figured you guys had put it there for a reason, so I was simply make all three functions across all three interfaces the same.

Do you want me to remove this "cast" from all 9 functions?